packages/next/src/build/index.ts

```diff
@@ -1260,6 +1260,9 @@ export default async function build(
       )
 
       const isAppDynamicIOEnabled = Boolean(config.experimental.dynamicIO)
+      const isNavigationEnabled = Boolean(
```

Comment:
this variable naming doesn't make much sense to me since navigation is always enabled, I think maybe it's missing a word